inc/audiences/namespace.php

```diff
@@ -755,3 +764,49 @@ function build_audience_query( array $audience ) : array {
 
 	return $groups_query;
 }
+
+/**
+ * Output audience configuration data as JSON.
```

Comment:
Doesnt look like this `output`s? `Get ` seems better maybe?

Reply:
Oh whoops, it's not meant to that should have been updated